src/app/clusters/descriptor/descriptor.cpp

```diff
@@ -124,7 +124,12 @@ CHIP_ERROR DescriptorAttrAccess::ReadDeviceAttribute(EndpointId endpoint, Attrib
 CHIP_ERROR DescriptorAttrAccess::ReadClientServerAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder, bool server)
 {
     CHIP_ERROR err = aEncoder.EncodeList([&endpoint, server](const auto & encoder) -> CHIP_ERROR {
-        uint16_t clusterCount = emberAfClusterCount(endpoint, server);
+        // TODO: emberAfClusterCount returns uint16_t yet emberAfGetNthCluster uses uint8_t
```

Comment:
No, emberAfClusterCount returns `uint8_t`.

Reply:
I was sure it did when I checked it ... ?!?! .... updating.